ui/src/containers/AlertHistoryProvider.spec.js

```diff
@@ -14,7 +14,13 @@ import StartTimeProvider from './StartTimeProvider';
 
 const wrapper = ({ children }) => (
   <MemoryRouter>
-    <QueryClientProvider client={new QueryClient()}>
+    <QueryClientProvider client={new QueryClient({
+      defaultOptions: {
+        queries: {
+          retry: false,
+        },
+      },   
```

Comment:
```suggestion
      },
```

Don't we have Prettier set up? :wink: